package.json

```
@@ -3,11 +3,12 @@
     "version": "1.0.4",
     "private": true,
     "scripts": {
-        "prestart": "yarn run buildplugin && yarn link docusaurus-plugin-navdata",
-        "buildplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && yarn link",
+        "prestart": "yarn run buildnavplugin && yarn run buildcookiesplugin",
+        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && (yarn unlink > /dev/null || true) && yarn link",
+        "buildcookiesplugin": "cd ./src/plugins/docusaurus-plugin-cookiesbanner && yarn install && (yarn unlink > /dev/null || true) && yarn link",
         "start": "docusaurus start --no-open",
-        "build": "yarn run buildplugin && yarn link docusaurus-plugin-navdata && docusaurus build",
-        "preinstall": "yarn run buildplugin && yarn link docusaurus-plugin-navdata",
+        "build": "yarn run buildnavplugin && yarn run buildcookiesplugin && docusaurus build",
+        "preinstall": "yarn run buildnavplugin && yarn run buildcookiesplugin",
```

[Build is failing](https://github.com/casper-network/docs-new/actions/runs/5242889749/jobs/9516993646?pr=207) with the following error:

```
Error:  Error: Docusaurus was unable to resolve the "docusaurus-plugin-navdata" plugin. Make sure one of the following packages are installed:
- docusaurus-plugin-navdata
```

That is because `yarn link docusaurus-plugin-navdata` was not called. Here is fixed version:


```suggestion
        "build": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner && docusaurus build",
        "preinstall": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner",
```